tests/docs/smoke-all/2025/02/01/11896.qmd

```diff
@@ -0,0 +1,18 @@
+---
+format: pptx
+_quarto:
+  tests:
+    pptx:
+      ensurePptxMaxSlides:
+        - 1
+      ensurePptxRegexMatches:
+        - '<a:t>Before<\/a:t><\/a:r><\/a:p><a:p><a:pPr lvl="0" indent="0" marL="0"><a:buNone \/><\/a:pPr><a:r><a:rPr \/><a:t>After<\/a:t>'
```

Comment:
You need to set the slide number in current API of `ensurePptxRegexMatches`

```suggestion
      ensurePptxRegexMatches:
        - ['<a:t>Before<\/a:t><\/a:r><\/a:p><a:p><a:pPr lvl="0" indent="0" marL="0"><a:buNone \/><\/a:pPr><a:r><a:rPr \/><a:t>After<\/a:t>']
        - 1
```

Reply:
Unnamed nested arrays 😅

Thanks Christophe, I did not even notice the path was no longer correct (too many tries between checking the codebase for the tests and trying the tests), that's why it stopped reporting issues in the test at some point.
A path check would be nice^^